pkg/api/team_members.go

```diff
@@ -30,8 +31,15 @@ func GetTeamMembers(c *m.ReqContext) Response {
 
 // POST /api/teams/:teamId/members
 func AddTeamMember(c *m.ReqContext, cmd m.AddTeamMemberCommand) Response {
-	cmd.TeamId = c.ParamsInt64(":teamId")
-	cmd.OrgId = c.OrgId
+	teamId := c.ParamsInt64(":teamId")
```

Comment:
Maybe we should revert this?

```suggestion
	cmd.TeamId = c.ParamsInt64(":teamId")
	cmd.OrgId = c.OrgId
```

Reply:
Not enough, but we can clean that up. On it.